crates/cxx-qt-gen/src/parser/qobject.rs

```diff
@@ -54,6 +56,27 @@ pub struct ParsedQObject {
 }
 
 impl ParsedQObject {
+    #[cfg(test)]
+    pub fn mock_parsed_qobject() -> Self {
```

Comment:
Why not just `mock()`? The function is already on ParsedQObject.

Reply:
Done, same for the structured obj